sdk/ai/azure-ai-projects/tests/agents/test_agents_client.py

```diff
@@ -73,14 +79,14 @@
 agentClientPreparer = functools.partial(
     EnvironmentVariableLoader,
     "azure_ai_projects",
-    azure_ai_projects_connection_string="https://foo.bar.some-domain.ms;00000000-0000-0000-0000-000000000000;rg-resour-cegr-oupfoo1;abcd-abcdabcdabcda-abcdefghijklm",
-    azure_ai_projects_data_path="azureml://subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-resour-cegr-oupfoo1/workspaces/abcd-abcdabcdabcda-abcdefghijklm/datastores/workspaceblobstore/paths/LocalUpload/000000000000/product_info_1.md",
+    azure_ai_projects_agents_tests_project_connection_string="region.api.azureml.ms;00000000-0000-0000-0000-000000000000;rg-resour-cegr-oupfoo1;abcd-abcdabcdabcda-abcdefghijklm",
+    azure_ai_projects_agents_tests_data_path="azureml://subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-resour-cegr-oupfoo1/workspaces/abcd-abcdabcdabcda-abcdefghijklm/datastores/workspaceblobstore/paths/LocalUpload/000000000000/product_info_1.md",
 )
 """
 agentClientPreparer = functools.partial(
     EnvironmentVariableLoader,
-    'azure_ai_project',
-    azure_ai_project_host_name="https://foo.bar.some-domain.ms",
+    'azure_ai_projects',
+    azure_ai_project_host_name="region.api.azureml.ms",
     azure_ai_project_subscription_id="00000000-0000-0000-0000-000000000000",
     azure_ai_project_resource_group_name="rg-resour-cegr-oupfoo1",
     azure_ai_project_workspace_name="abcd-abcdabcdabcda-abcdefghijklm",
```

Comment:
Why are the above 4 loaded separately? can't you just load the project connection string? Are those 4 used in other places?